### ISSUE TITLE
Restarting supervisorctl services failed with - django: ERROR (spawn error)

### ISSUE THREAD

**Issue by cws1121** [dimagi/cloudworks#2]
Posting on behalf of @MutaJosh

When restarting all the supervisorctl services, it's returning:
![spawn error](https://user-images.githubusercontent.com/16632752/131501072-cd83b79b-48db-464d-be60-1bde290bf7ac.png)

I had a look into logs, found that GUNICORN is not being able to start successfully [It couldn't create a socket] as below FYI
`ValueError: '/home/web/www/cloudworks/run/gunicorn.sock' is not a socket`

Also fails with 'Permission Denied' while serving that instance
![perm_denied](https://user-images.githubusercontent.com/16632752/131501166-ab43b3ca-1dbb-4e2b-b771-548562e7d37f.png)

Here are the log files:

[django.log](https://github.com/cws1121/cloudworks/files/7083854/django.log)
[gunicorn.log](https://github.com/cws1121/cloudworks/files/7083855/gunicorn.log)
[nginx-access.log](https://github.com/cws1121/cloudworks/files/7083856/nginx-access.log)
[nginx-error.log](https://github.com/cws1121/cloudworks/files/7083857/nginx-error.log)

**Comment by cws1121** [comment]
Hi Josue,

Seems like this is the same issue that was reported last time.
Again, the solution would be:

```
sudo rm /home/web/www/cloudworks/run/gunicorn.sock
sudo supervisorctl restart all
```

I would like to know why this issue occured again and I can’t replicate it in my test environment. Can you please try this:

```
cd /home/web/www/cloudworks/run
ls -al
```

**Comment by cws1121** [comment]
Posting on behalf of @MutaJosh

Thank you for that. This error kept arising;
The django app would start successfully, but gunicorn.sock in `cd /home/web/www/cloudworks/run` not created.
I tried this a couple of times, deleting the file and restarting supervisor service failing too, but worked on the fourth time.
Also, I had to change ownership (recursively) on `cd /home/web/www/cloudworks/folder `to user web, and it finally worked successfully. [This and the above points are not caught in docs😊]
